### PR TITLE
e2e tests for header component features

### PR DESCRIPTION
**Pull request by AdiMakkar**
1. Home Button
2. Network Button
3. Groups Button 
4. Jobs Button 
5. Support Button
6. Search Feature 
7. Messages Feature 
8. Notifications Feature 
9. Profile Feature 
10. Language Feature

### CODE REVIEW

**Comment by AdiMakkar** [comment]
This PR doesn't include directing confirmation of the various pages, since they haven't been developed yet. Comments are left to ensure once they are active, tests will verify the existence of these pages and not just the clickability of the button.

File: haibun-e2e-tests/local/backgrounds/pages/search.feature
Comment (vid) [inline haibun-e2e-tests/local/backgrounds/pages/search.feature:1]: This should either be generalized to all pages, or separate feature files created for each page type (main, messages, etc). There probably aren't enough pages yet to have many, so maybe just rename it "site.feature."

File: haibun-e2e-tests/local/features/header /groups-button.feature
Comment (vid) [inline haibun-e2e-tests/local/features/header /groups-button.feature:1]: Will need a final step to verify the user has landed on the right page.

File: haibun-e2e-tests/local/features/header /language-change.feature
Comment (vid) [inline haibun-e2e-tests/local/features/header /language-change.feature:1]: Need a test to verify the language has changed.

File: haibun-e2e-tests/package.json
Comment (vid) [inline haibun-e2e-tests/package.json:1]: should get rid of the scripts (eg test-xss) and modules (eg @haibun/web-server-express ) that aren't used by this repo.

Comment (AdiMakkar) [reply haibun-e2e-tests/package.json:1]: Gotten rid of scripts and modules not being used by this repo.

**Review by doug0102 — APPROVED** [review]

**Review by doug0102 — DISMISSED** [review]

**Pull request closed without merging**